DisCatSharp.Docs/articles/misc/hosting.md

```diff
@@ -22,6 +22,65 @@ Any modern hardware will work just fine, new or used.
 
 Depending on how complex your bot is, you may even consider purchasing a Raspberry Pi ($35).
 
+# Termux
+
+For those who don't have some PC or other hardware lying around alternatively you could use your phone. Using [Termux](https://termux.dev/en/) and a package called [proot-distro](https://github.com/termux/proot-distro), which we use to install a Debian vm and setup DotNET to run the bot. To those interested the steps are given below:
```

Comment:
> don't have some PC -> don't have a PC

Second sentence is kind of a mess, please rephrase.

Reply:
Alright